Library/Homebrew/extend/pathname.rb

```diff
@@ -29,7 +29,7 @@ def compute_disk_usage
     if self.directory?
       @file_count = 0
       @disk_usage = 0
-      self.find do |f|
+      find do |f|
```

Comment:
Oh, sorry, I use "call" as a pretty wide simple reference rather than an explicit definition. I appreciate that's probably something of a nightmare when discussing these things.